src/server/repository/public/ddl/getCreatePublicSchemaDDL.ts

```diff
@@ -0,0 +1,162 @@
+      id bigserial primary key,
+      assessment_id bigint not null,
+      uuid uuid default uuid_generate_v4(),
+      name character varying default '',
+      published boolean not null default false,
+      foreign key (assessment_id) references ${schemaName}.assessment (id)
+        on update cascade on delete cascade
+    );
+    create unique index if not exists assessment_cycle_uuid_key on ${schemaName}.assessment_cycle using btree (uuid);
+
+    create table if not exists ${schemaName}.country (
+      country_iso character varying(3) primary key not null,
+      config jsonb not null default '{}'::jsonb
+    );
+
+    create table if not exists ${schemaName}.file (
+      id bigserial primary key,
+      uuid uuid not null default uuid_generate_v4(),
+      name character varying(255) not null,
+      file bytea not null, -- TODO: Remove this when S3 branch merged
+      created_at timestamp without time zone not null default now()
+    );
+    create unique index if not exists file_uuid_key on ${schemaName}.file using btree (uuid);
+
+    create table if not exists ${schemaName}.users (
+      id bigserial primary key,
+      status ${schemaName}.users_status not null default 'invitationPending'::${schemaName}.users_status,
+      email character varying(255) not null,
+      props jsonb not null default '{}'::jsonb,
+      uuid uuid default uuid_generate_v4(),
+      profile_picture_file_uuid uuid,
+      foreign key (profile_picture_file_uuid) references ${schemaName}.file (uuid)
+        on update no action on delete no action
+    );
+    create unique index if not exists users_email_key on ${schemaName}.users using btree (email);
+
+    create table if not exists ${schemaName}.activity_log (
+      id bigserial primary key,
+      time timestamp without time zone not null default timezone('UTC'::text, now()),
+      message text not null,
+      country_iso character varying(3),
+      section character varying(250) not null,
+      target jsonb,
+      user_id bigint not null,
+      assessment_uuid uuid,
+      cycle_uuid uuid,
+      foreign key (assessment_uuid) references ${schemaName}.assessment (uuid)
+        on update cascade on delete cascade,
+      foreign key (country_iso) references ${schemaName}.country (country_iso)
+        on update no action on delete no action,
+      foreign key (cycle_uuid) references ${schemaName}.assessment_cycle (uuid)
+        on update cascade on delete cascade,
+      foreign key (user_id) references ${schemaName}.users (id)
+        on update cascade on delete cascade
+    );
+    create index if not exists idx_activity_log_cycle_message on ${schemaName}.activity_log using btree (cycle_uuid, message);
+    create index if not exists idx_activity_log_filtering on ${schemaName}.activity_log using btree (assessment_uuid, cycle_uuid, country_iso, message, time);
+
+    create table if not exists ${schemaName}.migration_steps (
```

Comment:
Not sure if this should be here, or should the script check and create if not exists?